lib/nerves/release.ex

```diff
@@ -6,6 +6,14 @@ defmodule Nerves.Release do
   def init(%{options: options} = release) do
     opts = Keyword.merge(options, release_opts())
 
+    vm_args = Mix.Release.rel_templates_path(release, "vm.args.eex")
```

Comment:
We should check the release options to see if this option exists instead of calling directly to `Mix.Release.rel_templates_path` since that is elixir 1.11+

Reply:
If the check below is no longer needed this line is obsolete anyways.